packages/driver/src/cypress/cy.ts

```diff
@@ -118,58 +120,284 @@ const setTopOnError = function (Cypress, cy) {
 
 // NOTE: this makes the cy object an instance
 // TODO: refactor the 'create' method below into this class
```

Comment:
TO DO is done? 😄

Reply:
Not yet. This is the first step to remove it. I'll remove it in the next PR.